### DISCUSSION TITLE
三星槓台積 3奈米要投產了

### DISCUSSION THREAD

**Discussion by Joan0518** [1101-GS3073-C/Python-Community-GS3073-C#386] [科技新知]
業界人士分析，三星雖然宣稱3奈米邁入量產倒數計時，但從電晶體密度、效能等層面分析，三星的3奈米與應與台積電5奈米家族的4奈米，以及英特爾的Intel 4製程相當，換言之，三星以「3奈米」為最新製程進度命名，表面上贏了面子，但在電晶體密度、效能等都還是落後台積電，「實際上還是輸了裡子」。
[新聞內容](https://udn.com/news/story/7240/6281268)
不知三星這波3奈米品質如何
而台積電也預計今年下半年3奈米量產!

**Comment by luo910227** [comment]
文中也提到台積電已在做更先進的2奈米布局，我認為三星所謂的倒數計時也只是虛張聲勢，台積電護國神山在此產業的領先狀況短時間是很難超越的
